testhelpers/core.go

```diff
@@ -184,3 +187,158 @@ func RequireCreateStorages(ctx context.Context, t *testing.T) (state.Tree, vm.St
 
 	return st, vms
 }
+
+type testStorage struct {
+	state interface{}
+}
+
+var _ exec.Storage = testStorage{}
+
+func (ts testStorage) Put(v interface{}) (cid.Cid, error) {
+	return cid.Cid{}, nil
+
+}
+func (ts testStorage) Get(cid.Cid) ([]byte, error) {
+	node, err := cbor.WrapObject(ts.state, types.DefaultHashFunction, -1)
+	if err != nil {
+		return []byte{}, err
+	}
+
+	return node.RawData(), nil
+}
```

Comment:
Please document the intent of these fake methods. There's nontrivial code here so it  looks like `Get` is doing something, but `Put` isn't? How is the fake behaviour supposed to work, and is this interim or more or less complete?